src/BizHawk.Emulation.Cores/Consoles/SNK/DualNeoGeoPort.cs

```diff
@@ -28,9 +28,10 @@ public class DualNeoGeoPort : IEmulator
 		[CoreConstructor("DNGP")]
 		public DualNeoGeoPort(CoreComm comm, byte[] rom, bool deterministic)
 		{
+			// this will throw on construct:  we need to compile two different copies of ngp at different starts

```

Comment:
The comment is wrong, since we did change the start parameter.  Dunno what it will do, just deadlock or something?  Maybe the core should be axed entirely.

Reply:
Broke off into another issue